core/lib/mcxtx.h

```diff
@@ -275,4 +275,6 @@
          */
         dr_zmm_t ymm[MCXT_NUM_SIMD_SLOTS];
     };
+    /** Storage for #MCXT_NUM_OPMASK_SLOTS mask registers as part of AVX-512. */
+    dr_opmask_t opmask[MCXT_NUM_OPMASK_SLOTS];
```

Comment:
This size change is going to break binary compatibility: the assert about the struct size not matching DR's is going to fire.  It may be best to go update that right now in this PR.

Reply:
Done, hopefully as expected. PTAL.